### DISCUSSION TITLE
Github Copilot not working on WSL for Windows?

### DISCUSSION THREAD

**Discussion by r1g023** [community/community#8437] [Copilot Conversations] [answered]
Hi, I recently tried using the copilot extension but I get an error when I try to authenticate the addon. 

[INFO] [auth] [2021-12-04T20:25:37.825Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2021-12-04T20:25:37.830Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

Any ideas? Or it seems like I might have to wait until further notice? 

I also did try it on VS Code without using a remote session of WSL and I got the same error.

**Comment by davecheney** [comment, marked as answer]
The error indicates that you are not part of the technical preview. Did you sign up for an invite?

**Reply by r1g023** [reply]
Yes I signed up for an invite but that was only yesterday when I signed up after downloading the extension. Perhaps I have to wait a little longer in that case?

**Reply by davecheney** [reply]
If you have not received an email that your were accepted into technical preview then you are most likely waitlisted. We are adding people as we have capacity. Please be patient.
